src/pfe/file-watcher/server/test/functional-test/suites/project/tests/project-specification.ts

```diff
@@ -322,6 +325,61 @@ export function projectSpecificationTest(socket: SocketIO, projData: projectsCon
             }
         }).timeout(timeoutConfigs.defaultTimeout);
 
+        it(`set internalPort to ${constants.disablePingPort} in .cw-settings file to check for serverless app`, async () => {
+            let events, eventDatas;
+
+            events = [eventConfigs.events.settingsChanged, eventConfigs.events.statusChanged];
+            eventDatas = [{
+                "projectID": projData.projectID,
+                "status": "success",
+                "ports": {
+                    "internalPort": `${constants.disablePingPort}`,
+                }
+            }, {"projectID": projData.projectID, "appStatus": "stopped"}];
+            await reconfigProjectSettings("internalPort", constants.disablePingPort, events, eventDatas);
+
+            const projectInfo = await projectUtil.getProjectInfo(projData.projectID);
+            await projectUtil.getContainerInfo(projectInfo, true);
```

Comment:
is this required here? Not using it ..

Reply:
We do this to update the latest container info....notice how we pass the `true` in the second arg which means force refresh and have the latest container info. We do similar things [here](https://github.com/eclipse/codewind/blob/74c6f4f4f7d6608a3ad1c018dd239bd8ca517852/src/pfe/file-watcher/server/test/functional-test/suites/project/tests/project-specification.ts#L458)